.travis.yml

```diff
@@ -1,22 +1,35 @@
 sudo: false
 language: python
-python:
-  - "2.7"
-  - "3.6"
+
 cache: pip
+before_install:
+  - pip install --upgrade pip
 install:
   - pip install -r requirements-dev.txt
   - pip install coveralls
 script:
-  - DJANGO_SETTINGS_MODULE=test_settings coverage run --source='google_analytics' `which django-admin.py` test
+  - DJANGO_SETTINGS_MODULE=test_settings coverage run --source='google_analytics' "$(which django-admin)" test
   - flake8
 after_success:
   - coveralls
-deploy:
-  provider: pypi
-  user: Praekelt
-  password:
-    secure: O0feip8iJtd8Pulu2m9qcAg2yqwBUe5FsvYSaWTJke8a9oiS4xGittATKqbxUZNvFjHpbS79V2032IPDs+oNePwotSavPiKeSF1xNxAaeGuEH7nFuaYQ4/RzLpHHYU3KT61sa32U+bdH3AoDvAb/5MzlLucTVGLEuCUpHXLhcx0=
-  on:
-    tags: true
-    all_branches: true
+
+jobs:
+  include:
+    - python: '2.7'
+    - python: '3.4'
+    - python: '3.5'
+    - python: '3.6'
+
+    - stage: release
```

Comment:
This is necessary because the new pypi.org won't accept more than one upload per release, so we can't do a deploy on every build we run like we used to.